src/app/components/offeringCard/offeringCard.tsx

```diff
@@ -14,7 +16,7 @@ export default function OfferingCard({
   return (
     <div className={styles.offeringCard}>
       <div className={styles.textContainer}>
-        <p className={styles.text}>Want to become a speaker at an event?</p>
+        <p className={styles.text}>{text || defaultText}</p>
```

Comment:
there doesn't need to be any defaultText. Update to `{text && <p className={styles.text}>{text}</p>`

Reply:
Removed the defaultText value and updated line:17